rs/execution_environment/src/canister_manager/tests.rs

```diff
@@ -1915,205 +1636,19 @@ fn add_cycles_sender_not_in_whitelist() {
-            canister_id,
-            UNIVERSAL_CANISTER_WASM.to_vec(),
-            vec![],
-            None,
-            Some(memory_allocation),
-        ))
-        .unwrap_err();
-    assert_eq!(err.code(), ErrorCode::InsufficientMemoryAllocation);
-    assert!(err.description().contains(&format!(
-        "Canister was given {} B memory allocation but at least",
-        memory_allocation,
-    )));
-}
-
-#[test]
-fn upgrading_canister_fails_if_memory_capacity_exceeded() {
-    let initial_cycles = Cycles::new(1_000_000_000_000_000);
-    let mb = 1 << 20;
-    let memory_capacity = 1000 * mb;
-    let memory_used = memory_capacity - 10 * mb;
-
-    let wat = r#"
-        (module
-            (import "ic0" "stable64_grow" (func $stable64_grow (param i64) (result i64)))
-            (func (export "canister_pre_upgrade")
-                (drop (call $stable64_grow (i64.const 80)))
-            )
-            (func (export "canister_post_upgrade")
-                (drop (call $stable64_grow (i64.const 80)))
-            )
-            (memory 0)
-        )"#;
-
-    let mut test = ExecutionTestBuilder::new()
-        .with_subnet_execution_memory(memory_capacity as i64)
-        .with_subnet_memory_reservation(0)
-        .build();
-
-    let wasm = wat::parse_str(wat).unwrap();
-
-    let canister1 = test.create_canister(initial_cycles);
-    let canister2 = test.create_canister(initial_cycles);
-
-    test.install_canister_with_allocation(canister1, wasm.clone(), None, Some(memory_used))
-        .unwrap();
-
-    test.install_canister_with_allocation(canister2, wasm.clone(), None, None)
-        .unwrap();
-
-    let cycles_before = test.canister_state(canister2).system_state.balance();
-    let execution_cost_before = test.canister_execution_cost(canister2);
-    test.upgrade_canister_with_allocation(canister2, wasm.clone(), None, Some(11 * mb))
-        .unwrap_err()
-        .assert_contains(
-            ErrorCode::SubnetOversubscribed,
-            "Canister requested 11.00 MiB of memory but only 10.00 MiB are \
-            available in the subnet.",
-        );
-
-    assert_eq!(
-        test.canister_state(canister2).system_state.balance(),
-        cycles_before - (test.canister_execution_cost(canister2) - execution_cost_before),
-    );
-
-    // Try upgrading without any memory allocation.
-    test.upgrade_canister_with_allocation(canister2, wasm, None, None)
-        .unwrap_err()
-        .assert_contains(
-            ErrorCode::SubnetOversubscribed,
-            "Canister requested 10.00 MiB of memory but only 10.00 MiB are available \
-        in the subnet.",
-        );
-    let execution_cost_after = test.canister_execution_cost(canister2);
-
-    assert_eq!(
-        test.canister_state(canister2).system_state.balance(),
-        cycles_before - (execution_cost_after - execution_cost_before)
-    );
-}
-
-#[test]
-fn installing_a_canister_with_not_enough_cycles_fails() {
```

Comment:
Why is this test removed?

Reply:
Hmm, bad merge I think, definitely should not be removed.